ResultPicker.js

```diff
@@ -1,49 +1,53 @@
-// modified version of https://superuser.com/a/1235058/69589
-
-// find all the search results
-const querySelector = 'div.yuRUbf';
+// from https://github.com/RobinH-J/SearchingShortcuts
 
+// selector to find all Google search results. Simplified Google html:
+//   <div class="yuRUbf">
+//     <a href="The real url is here">
+//       <br>
+//       <h3>The big text we see in Google Search results</h3>
+//     </a>
+//   </div>
+// And sometimes we get pseudo-results (like Ads) that don't contain the h3: exclude them.
+const querySelector = 'div.yuRUbf h3';
```

Comment:
this only changed with the new `h3` appended on the end, which explicitly excludes Ads (though Ads weren't a problem before, if i remember correctly)